bin/validate_man_lastnight_fa

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# requires source /project/projectdirs/desi/software/desi_environment.sh master
+from astropy.table import Table
+import numpy as np
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--night", help="use this if you want to specify the night, rather than just use the last one",default='20210921')
+parser.add_argument("--clean", help="remove all files in the output directory prior to start, True/Fase",default=False)
+args = parser.parse_args()
+
+month = args.night[:6]
+
+#open exposures file
+exps = Table.read('/global/cfs/cdirs/desi/spectro/redux/daily/exposure_tables/'+month+'/exposure_table_'+args.night+'.csv')
+
+#check if tileid are in main
+tlm = Table.read('/global/cfs/cdirs/desi/survey/ops/surveyops/trunk/ops/tiles-main.ecsv')
```

Comment:
Let's use something like `desispec.io.specprod_root()` or `os.path.expandvars('$DESI_SPECTRO_REDUX/$SPECPROD')` or `os.path.expandvars('$DESI_ROOT/spectro/redux/daily/ ...)` so that we're not hardcoding a NERSC-specific path.

Expanding on that: for debugging there may be times when we run to run off of something other than "daily"; let's upgrade that to a `--specprod` option with the default of "daily".